tests/scripts/analyze_outcomes.py

```diff
@@ -51,6 +53,25 @@ def hits(self):
         """
         return len(self.successes) + len(self.failures)
 
+def execute_reference_driver_tests(ref_component, driver_component, outcome_file):
+    """Run the tests that will fullfill the outcome file used for the following
```

Comment:
Minor:
```suggestion
    """Run the tests that will create the outcome file used for the following
```
fulfil is not the right word here, as it's more about duties or achievements, not literal "filling out" :)